templates/js/routes/index-static.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET home page. */
+
+router.get('/', function(req, res, next) {
+  res.sendfile('public/index.html');
```

Comment:
Do not duplicate this entire file for static support, because now we have to maintain two almost identical files. Please adjust the existing file to support both with template strings like we do already.